docs/src/tasks.md

```diff
@@ -0,0 +1,21 @@
+# Tasks
+
+Zed supports ways to spawn (and rerun) commands using its integrated terminal to output the results.
+
+Currently, two kinds of tasks are supposed, but more will be added in the future.
+
+## Static tasks
+
+Tasks, defined in a config file (`tasks.json` in the Zed config directory) that do not depend on the current editor or its content.
+
+Config file can be opened with `zed::OpenTasks` action ("zed: open tasks" in the command pane), it will have a configuration example with all options commented.
```

Comment:
```suggestion
Config file can be opened with `zed::OpenTasks` action ("zed: open tasks" in the command palette), it will have a configuration example with all options commented.
```